tests/21-ct-clean-up.sh

```diff
@@ -0,0 +1,212 @@
+}
+
+log "beginning connectivity test with BIDIRECTIONAL=${BIDIRECTIONAL}"
+monitor_clear
+
+test_reachability "client" "[$SERVER_IP]"
+test_reachability "client-2" "[$SERVER_IP]"
+monitor_clear
+
+test_reachability "client" "$SERVER_IP4"
+test_reachability "client-2" "$SERVER_IP4"
+monitor_clear
+
+test_reachability "client" "[$SERVER_2_IP]"
+test_reachability "client-2" "[$SERVER_2_IP]"
+monitor_clear
+
+test_reachability "client" "$SERVER_2_IP4"
+test_reachability "client-2" "$SERVER_2_IP4"
+monitor_clear
+
+log "connectivity_test complete"
+
+sudo cilium bpf ct list global
+
+entriesBefore=$(sudo cilium bpf ct list global | wc -l)
+
+bef_client_server_2_ct_entries=$(count_ct_entries_of "\[${SERVER_2_IP}\]" "\[${CLIENT_IP}\]" "${CLIENT_SEC_ID}")
+bef_client4_server_2_ct_entries=$(count_ct_entries_of "${SERVER_2_IP4}" "${CLIENT_IP4}" "${CLIENT_SEC_ID}")
+bef_client_2_server_2_ct_entries=$(count_ct_entries_of "\[${SERVER_2_IP}\]" "\[${CLIENT_IP}\]" "${CLIENT_SEC_ID}")
+bef_client4_2_server_2_ct_entries=$(count_ct_entries_of "${SERVER_2_IP4}" "${CLIENT_2_IP4}" "${CLIENT_SEC_ID}")
+
+policy_delete_and_wait id=server-2
+
+wait_for_endpoints 4
+
+aft_client_server_2_ct_entries=$(count_ct_entries_of "\[${SERVER_2_IP}\]" "\[${CLIENT_IP}\]" "${CLIENT_SEC_ID}")
+aft_client4_server_2_ct_entries=$(count_ct_entries_of "${SERVER_2_IP4}" "${CLIENT_IP4}" "${CLIENT_SEC_ID}")
+aft_client_2_server_2_ct_entries=$(count_ct_entries_of "\[${SERVER_2_IP}\]" "\[${CLIENT_IP}\]" "${CLIENT_SEC_ID}")
+aft_client4_2_server_2_ct_entries=$(count_ct_entries_of "${SERVER_2_IP4}" "${CLIENT_2_IP4}" "${CLIENT_SEC_ID}")
+
+echo "bef_client_server_2_ct_entries $bef_client_server_2_ct_entries"
+echo "bef_client4_server_2_ct_entries $bef_client4_server_2_ct_entries"
+echo "bef_client_2_server_2_ct_entries $bef_client_2_server_2_ct_entries"
+echo "bef_client4_2_server_2_ct_entries $bef_client4_2_server_2_ct_entries"
+echo "aft_client_server_2_ct_entries $aft_client_server_2_ct_entries"
+echo "aft_client4_server_2_ct_entries $aft_client4_server_2_ct_entries"
+echo "aft_client_2_server_2_ct_entries $aft_client_2_server_2_ct_entries"
+echo "aft_client4_2_server_2_ct_entries $aft_client4_2_server_2_ct_entries"
+
+entriesAfter=$(sudo cilium bpf ct list global | wc -l)
+
+if [ "$(( entriesBefore - entriesAfter ))" -ne "8" ]; then
+    abort "CT map should have exactly 8 entries less after deleting the policy"
+fi
+
+test_reachability "client" "[$SERVER_IP]"
+test_reachability "client-2" "[$SERVER_IP]"
+monitor_clear
+
+test_reachability "client" "$SERVER_IP4"
+test_reachability "client-2" "$SERVER_IP4"
+monitor_clear
+
+test_unreachability "client" "[$SERVER_2_IP]"
+test_unreachability "client-2" "[$SERVER_2_IP]"
+monitor_clear
+
+test_unreachability "client" "$SERVER_2_IP4"
+test_unreachability "client-2" "$SERVER_2_IP4"
+monitor_clear
+
+log "deleting all policies in Cilium"
+cilium policy delete --all
+
+test_succeeded "${TEST_NAME}"
```

Comment:
Please add a ticket to migrate this test over to the Ginkgo CI.

Reply:
Done #1916